src/inspect_evals/swe_bench/tests/create_test_repos.py

```diff
@@ -2,38 +2,43 @@
 import os
 from pathlib import Path
 
-import pandas as pd
+import pandas as pd  # type: ignore
 from datasets import load_dataset
 
 dataset = load_dataset("princeton-nlp/SWE-bench_Verified")["test"]
 
 # We  create a subset of SWE-bench-verfied which:
 # 1) Contains all of the repositories in SWE-bench verified
 # 2) For each repository, contains an example where swe_agent + Claude 3.5 + Sonnet resolved the issue, and an example where it did not
+# This is used to test that our version of swe_bench has the same behaviour as the original version.
 
 results_per_repo = {}
-baseline_dir = Path(__file__).parent.parent / "baselines"
-logs_dir = baseline_dir / "20240620_sweagent_claude3.5sonnet" / "logs"
+swebench_baseline_logs_dir = (
```

Comment:
This isn't good b/c it will result in files written into the package directory. You should use `platformdirs` here to get access to one of the standard storage location (probably `user_cache_dir()`)

Reply:
One problem though is still that if for example I run these and then I do a build of the package for PyPI these files will end up in the package!  I think anything temporary or scratch should not be written into the package src dir.